src/render-prop/render-prop.js

```diff
@@ -2,16 +2,27 @@ const React = require('react');
 const omit = require('lodash.omit');
 const { shallow } = require('enzyme');
 
-const shallowRenderProp = (wrapper, expander) =>
-  typeof expander === 'string'
-    ? wrapper.prop(expander)()
-    : expander(wrapper.props());
+// A wrapper component so that we can shallow-render immediately
+const Wrapper = props => props.children;
+Wrapper.displayName = 'Wrapper';
```

Comment:
@tdeekens This was a way around using a `div`. So it should now work with react-native out of the box again :)

Reply:
We're using a "trick" here to work around `shallow()` "swallowing" the first component. We simply create another component around it as a wrapper, which only renders the children. That one is then shallow-rendered. We thus end up with the children only.

This is what we would expect most of the time, and why your original implementation was so nice to use!